src/styles/Profile.module.js

```diff
@@ -5,7 +5,117 @@ const PorfileWrapper = styled.div`
   margin: 15px auto;
   border: 1px solid red;
   display: flex;
+  flex-direction: column;
   justify-content: center;
+  align-items: center;
 `;
 
-export {PorfileWrapper};
+const UserImage = styled.div`
+  border-radius: 50px;
+  width: 100px;
+  height: 100px;
+  background-color: gray;
+  margin: 20px auto 0px auto;
+`;
+
+const UserName = styled.p`
+  font-size: 1.5rem;
+  margin: 15px auto 10px auto;
+  font-weight: bold;
+`;
+
+const ShowFollow = styled.p`
+  font-size: 1rem;
+  margin: 2px auto;
+`;
+
+const ButtonWeapper = styled.div`
```

Comment:
I guess it is ButtonWrapper...

Reply:
After updating, please use global search to double-check the misspelling issue if they are all resolved.